libs/forms/src/lib/components/form-generator/form-generator.component.ts

```diff
@@ -69,14 +71,27 @@ export class FormGeneratorComponent implements OnInit {
   /** Datafields for dynamic data */
   @Input() datafields?: FormsLib.Datafields | null = {};
   /** Disable submit button. Otherwise will rely on the form validators to allow submission */
-  @Input() disableSubmit = false;
+  @Input() disableSubmit: null | boolean = false;
+  /** Enable/disable the form  */
+  @Input() disabled?: null | boolean = false;
   /** When the user submits the form */
   @Output() completed = new EventEmitter<unknown>();
 
   constructor() {}
 
   ngOnInit(): void {}
 
+  ngOnChanges(changes: SimpleChanges): void {
+    // Disable/enable the form when the input changes
+    if (changes['disabled'] && this.formGroup && is.browser) {
```

Comment:
I see the `formModel` `@Input()` but how does `this.formGroup` get defined? Only reason I ask is because the `setTimeout()` is killing me inside and trying to remember how I solved this in the past.

Reply:
Same comment as above, separate lib